Remotion/Validation/Core/Validators/IMaximumLengthValidator.cs

```diff
@@ -18,10 +18,8 @@
 
 namespace Remotion.Validation.Validators
 {
-  public interface ILengthValidator : IPropertyValidator
+  public interface IMaximumLengthValidator : IPropertyValidator
   {
-    int Min { get; }
-
     int? Max { get; }
```

Comment:
Hmm... since we can no longer specify a nullable maxlength via ctor, we can (and should) also change the interface to use a not-nullable int for maxlength.